documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -152,11 +153,14 @@ polling, as it does not require Jenkins to query GitHub on regular intervals.
 
 ### GitHub PR Labels
 
-GitHub PRs also pass labels through to the Jenkins build system. These labels can be accessed via the `PR_LABELS` environment variable from within scripts. Currently, there are 3 labels of importance to the Jenkins system.
+GitHub PRs pass labels through to the Jenkins build system. These labels can be accessed via the `PR_LABELS` environment variable from within scripts. Currently, there are 3 labels of importance to the Jenkins system.
```

Comment:
"Currently, there are 3 labels of importance to the Jenkins system." needs rephrasing: there are three labels that affect the steps executed in the pipeline and one that controls what is built